wagtail/admin/views/tags.py

```diff
@@ -19,7 +19,7 @@ def autocomplete(request, app_name=None, model_name=None):
 
     term = request.GET.get("term", None)
     if term:
-        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")
+        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")[:10]
```

Comment:
I just noticed that we only need the `name`s of the tags. We can use `values_list("name", flat=True)` to optimise the query.

Reply:
Makes sense, thanks!